src/app/main/component/ubs/services/order.service.ts

```diff
@@ -124,4 +127,8 @@ export class OrderService {
     this.localStorageService.removeUbsOrderId();
     this.shareFormService.saveDataOnLocalStorage();
   }
+
+  getStepperFinal(data: boolean) {
```

Comment:
I believe it should be `setStepperOnFinalStep`

Reply:
done local. I'll test it and commit